conf.yaml

```
@@ -127,8 +127,8 @@ contents:
             prefix:     en/welcome-to-elastic
             current:    *stackcurrent
             index:      welcome-to-elastic/index.asciidoc
-            branches:   [ 8.4 ]
-            live:       [ 8.4 ]
+            branches:   [ 8.5 ]
```

This will make the 8.5 branch "live" only when the release is made. Since we only ever publish one version of this book, maybe we should stop using versioned branches and just stick with `main`. What do you think, @nrichers 

```suggestion
            branches:   [ *stackcurrent ]
```

---

We're actually going to start publishing multiple versions of this book.  I'll add a suggestion.